scorer/verifications/brightid.py

```diff
@@ -0,0 +1,26 @@
+import time
+from arango import ArangoClient
+
+
+def verify():
+    print('BrightID')
+    db = ArangoClient().db('_system')
+    for u in db['users']:
+        verifications = set([v['name'] for v in db['verifications'].find({'user': u['_id']})])
+        callJoined = 'CallJoined' in verifications
+        seedConnected = 'SeedConnected' in verifications
+        if not (u['score'] >= 3 or (callJoined and seedConnected)):
+            continue
+        if 'BrightID' not in verifications:
+            db['verifications'].insert({
+                'name': 'BrightID',
```

Comment:
I assume `name` is indexed in the `verifications` collection

Reply:
It is indexed now
https://github.com/BrightID/BrightID-Node/commit/71c2d5464399b62979dceb0c08a419262333dd98